mantidproject/mantid
# Replacing MaxEnt with MuonMaxent in the frequency domain analysis GUI

The frequency domain analysis (FDA) GUI was requested to fit a single frequency spectrum to all of the time domain spectra. So we have used the replaced MaxEnt with the MuonMaxent algorithm. 

**To test:**
1. Open muon analysis and load some data (MUSR00062260 is nice)
2. Open the frequency domain analysis (you can then close muon analysis)
3. Switch to the MaxEnt page by changing the top drop down menu
4. Click the Calculate button - and a frequency spectrum will be added to the grouped workspace
5. If you select the outputs then these will also be added to the grouped workspace with appropriate names
6. Select Use Phase Table and try to run it (it should fail and give an error message)
7. Change the workspace to the run number (instead of MuonAnalysisGrouped_1) and run the code
8. A table workspace (PhaseTable) should now exist. Open it and change the first phase to 5.
9. In the GUI untick the Construct Phase Table option
10. Tick the Fix phases and the Output phase table options
11. Click the calculate button
12. Check the output phase table has the same (changed) values as the original phase table (i.e. is the first phase 5).



Fixes #21765. 

**Release Notes** 

added to release notes and doc updated
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
